packages/common/package.json

```diff
@@ -94,7 +94,7 @@
     "@solana/spl-token": "0.3.8",
     "@solana/web3.js": "1.93.4",
     "@stripe/crypto": "0.0.4",
-    "react": "^18.3.1",
+    "react": "18.3.1",
```

Comment:
Did you uninstall/reinstall the ones you pinned? Just want to make sure the package-lock got reverted to the pinned versions if we were a minor/patch ahead of them

Reply:
yea, i ran npm i afterwards